WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -156,6 +156,13 @@ public void onCreate() {
         HelpshiftHelper.init(this);
         initWpDb();
 
+        // EventBus setup
+        EventBus.TAG = "WordPressEvent";
+        EventBus.builder()
+                .logNoSubscriberMessages(false)
+                .sendNoSubscriberEvent(false)
+                .installDefaultEventBus();
+
```

Comment:
What about using `.throwSubscriberException(true)` (or at least `.throwSubscriberException(PackageUtils.isDebugBuild())`). I prefer the app to crash (even on production, we're good at fixing them).